sormas-api/src/main/resources/strings.properties

```diff
@@ -755,6 +756,7 @@ infoPickOrCreateSuperordinateEventForEvent = The list below contains all existin
 infoPickOrCreateEventGroupForEvent = The list below contains all existing event groups. Please check whether the event group this event belongs to is already on this list or create a new one if it isn't.
 infoSelectOrCreatePersonForCase = The database already contains at least one person that seems to be very similar to the personal details of the created case.<br/><br/>Please look through the list of persons. If you feel certain that one of those persons matches your case person, select them and click on the <i>Save</i> button. Otherwise, click on <i>Create New Person</i> to create a new person for your case.<br/><br/>If you are unsure, you can discard this window and cancel the case creation process.
 infoSelectOrCreatePersonForCaseImport = The database already contains at least one person that seems to be very similar to the personal details of the imported case.<br/><br/>Please look through the list of persons. If you feel certain that one of those persons matches your case person, select them and click on the <i>Save</i> button. Otherwise, click on <i>Create New Person</i> to create a new person for your case.<br/><br/>If you are unsure, you can discard this window to remove the case from the current import.
+infoSelectOrCreatePersonForTravelEntryImport = The database already contains at least one person that seems to be very similar to the personal details of the imported travel entry.<br/><br/>Please look through the list of persons. If you feel certain that one of those persons matches your travel entry person, select them and click on the <i>Save</i> button. Otherwise, click on <i>Create New Person</i> to create a new person for your travel entry.<br/><br/>If you are unsure, you can discard this window to remove the travel entry from the current import.
```

Comment:
I'm not sure about that. Making it generic means that we need a placeholder for the entity name. That, in turn, means that we need capitalised and uncapitalised strings for these entity names, but which of those are used depends on the language, i.e. we have no way to do this reliably in the code. The result will, in any case, be worse than having it properly translated, even if that means a little more effort for the translators.

Reply:
Wouldn't use a placeholder. Instead:

> "The database already contains at least one person that seems to be very similar to the personal details of the imported entry.<br/><br/>Please look through the list of persons. If you feel certain that one of those persons matches the entry's person, select it and click on the <i>Save</i> button. Otherwise, click on <i>Create New Person</i> to create a new person.<br/><br/>If you are unsure, you can discard this window to remove the entry from the current import."

We can also keep it as is.